index.js

```diff
@@ -44,8 +44,8 @@ module.exports = class Kernel {
   //       otherwise execution succeeded
   codeHandler (code, ethInterface = new Interface(new Environment())) {
     const debugInterface = new DebugInterface(ethInterface.environment)
-
-    const instance = Wasm.instantiateModule(code, {
```

Comment:
Can you make it work with both ways? i.e. check for the presence of `instantiateModule.`